apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -17,6 +17,13 @@ function should_show_coming_soon_page() {
 		return false;
 	}
 
+	// mShots should be able to take screenshots of coming soon sites, in the
+	// same way RSS readers have access to coming soon sites.
+	// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	if ( isset( $_GET['mshots'] ) ) {
```

Comment:
```suggestion
	if ( is_front_page() && isset( $_GET['mshots'] ) ) {
```

I'm using `is_front_page` instead of `is_home`, because the front page could be the main blog with the latest posts, instead of a static page.